openapi/components/schemas/SubscriptionChange.yaml

```
@@ -45,3 +45,9 @@ properties:
       to preview the changes that would be made to a subscription.
     type: boolean
     default: false
+  keepTrial:
+    description: >-
+      If set to true and the old plan has an active trial, that trial will be used for the new plan. Works with 'retain'
```

This API should have no effect on plans.
```suggestion
      If set to true and the subscription order has an active trial, it will use that trial further. Works with 'retain'
```